cmd/utils.go

```diff
@@ -636,6 +638,26 @@ func getGitLabels(dryrun bool) (map[string]string, error) {
 		}
 	}
 
+	if commitInfo.Author != "" && commitInfo.AuthorEmail != "" {
+		labels[appsodyImageCommitKeyPrefix+"author"] = commitInfo.Author + " <" + commitInfo.AuthorEmail + ">"
+	}
+
+	if commitInfo.Committer != "" && commitInfo.CommitterEmail != "" {
+		labels[appsodyImageCommitKeyPrefix+"committer"] = commitInfo.Committer + " <" + commitInfo.CommitterEmail + ">"
+	}
+
+	if commitInfo.Date != "" {
+		labels[appsodyImageCommitKeyPrefix+"date"] = commitInfo.Date
+	}
+
+	if commitInfo.Message != "" {
+		labels[appsodyImageCommitKeyPrefix+"message"] = commitInfo.Message
+	}
+
+	if commitInfo.contextDir != "" {
```

Comment:
Did you figure out the default value from S2I? Do they just not include this label if the project is at the root?

Reply:
These both suggest that we are overriding the default behaviour:
https://github.com/openshift/source-to-image/blob/master/docs/cli.md#context-directory
https://docs.openshift.com/container-platform/3.3/dev_guide/builds.html#source-code
So my intuition says that it should be blank?

Reply:
Ok, we will not include the contextDir label if the appsody project is at the root of the git repo